tests/libgstc/test_libgstc_pipeline_list_properties.c

```diff
@@ -134,14 +135,14 @@ gstc_json_child_string (const char * json, const char * parent_name,
 GST_START_TEST (test_pipeline_list_properties_success)
 {
   GstcStatus ret;
-  char *expected = "read /pipelines/pipe/elements/element/properties";
-  char *pipeline_name = "pipe";
-  char *element_name = "element";
+  const char *expected = "read /pipelines/pipe/elements/element/properties";
+  const char *pipeline_name = "pipe";
+  const char *element_name = "element";
   char **response;
   int array_lenght;
 
   ret =
-      gstc_element_properties_list (_client, pipeline_name, element_name,
+      gstc_element_properties_list (_client, (gchar*)pipeline_name, (gchar*)element_name,
```

Comment:
Same, see if `gstc_element_properties_list` can receive a `const gchar *` instead

Reply:
Yes, but if it is changed, also must be changed 'GstdClientCmd' array and also all the command functions (gstd_client_cmd_*) to keep coherent the code. I think those changes must be included as a refactor of this functions, since the change affects other parts of the code.